pkgs/applications/networking/browsers/geopard/default.nix

```diff
@@ -0,0 +1,72 @@
+{ stdenv
+, glib
+, gtk4
+, libadwaita
+, pango
+, rustPlatform
+, openssl
+, pkg-config
+, lib
+, wrapGAppsHook4
+, meson
+, ninja
+, desktop-file-utils
+, blueprint-compiler
+, appstream-glib
+, fetchFromGitHub
+}:
+
+stdenv.mkDerivation rec {
+  pname = "geopard";
+  version = "1.2.0";
+
+  cargoDeps = rustPlatform.importCargoLock {
+    lockFile = "${src}/Cargo.lock";
```

Comment:
I don't think this is allowed. the cargo.lock file needs to be placed next to the default.nix file.